### ISSUE TITLE
Add connection pooling for database on preview

### ISSUE THREAD

**Issue by bosko-m** [IntersectMBO/govtool#2342]
Database check (Preview / Database) is added to the health status page for GovTool on Preview.
https://status.gov.tools/status/preview-gov-tools

We need to add connection pooling in order to avoid scenario where health status page consume all db connection.

Consider using pgpool or pgbouncer.

**Comment by bosko-m** [comment]
CC @spannercode

**Comment by spannercode** [comment]
As I cannot contribute to the solution on this issue, I'm un-assigning myself.
So far, the preview environment seems stable on the health checks.

**Comment by bosko-m** [comment]
@spannercode we do need to take a look at intermittent failures though

**Comment by aaboyle878** [comment]
I agree with @stepa-ict we could potentially use RDS Proxy or an alternative could be to add Postgres as was discussed a while back however this might require some effort as qovery does not natively support a connection pooling solution.

However when setting up DB-Sync I allowed 200 max connections and looking at the activity dashboard on PGAdmin it only ever seems to use a max of 9 at any given time

**Comment by bosko-m** [comment]
Closing this one for the time being.
